lib/potluck.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'dish'
+
+class Potluck
+  attr_reader :date, :dishes, :menu
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    list = []
+    @dishes.map do |dish|
```

Comment:
When you are using #map, you do not need to shovel the mapped elements into a new array - map does this automatically.  You could refactor this method to look like this:

```ruby
def get_all_from_category(category)
  @dishes.map do |dish|
     dish if dish.category == category
  end
end
```